scripts/populate_user_join_tables.py

```diff
@@ -0,0 +1,55 @@
+from db import get_session
+
+import sqlalchemy as sql
+
+from models import (
+    Project,
+    Study,
+    ProjectUser,
+    StudyUser,
+)
+
+user_uuid = 'b5d6e7b6-b39c-4ffe-be22-84d58b6ee964'
```

Comment:
This just creates project-user and study-user links for this particular UUID I have on my local machine. We can "log in" as this user on prod as well or update this UUID to other ones to share access between projects.